src/components/Register.vue

```diff
@@ -83,7 +80,22 @@ export default class AddTicket extends Vue {
     repPassword: ""
   };
   onSubmit() {
-    console.log(this.form);
+    this.$store
+      .dispatch("ticketState/register", {
+        firstName: this.form.firstName,
+        lastName: this.form.lastName,
+        email: this.form.email,
+        department: this.form.department,
+        password: this.form.password,
+        repPassword: this.form.repPassword
+      })
+      .then(res => {
+        console.log(res);
+        if (res.status === 200) {
+          this.$router.push({ name: "signin" });
+        }
+      })
+      .catch(err => console.log(err));
```

Comment:
could you add some message when we catch an error 
https://element.eleme.io/#/en-US/component/notification